php-frontend/src/main/java/org/sonar/php/metrics/MetricsVisitor.java

```diff
@@ -126,12 +118,8 @@ private void setLineAndCommentMeasures(PhpFile file) {
     linesOfCode.forEach(lineOfCode -> fileLinesContext.setIntValue(CoreMetrics.NCLOC_DATA_KEY, lineOfCode, 1));
     commentLines.forEach(commentLine -> fileLinesContext.setIntValue(CoreMetrics.COMMENT_LINES_DATA_KEY, commentLine, 1));
 
-    if (saveExecutableLines) {
       executableLineVisitor.getExecutableLines().forEach(line ->
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this line start at column 5. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AIndentationCheck)